backEnd/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
```

Comment:
your node_modules got committed somehow, can you make sure that this line works?